packages/component-library/draft/Kaizen/SplitButton/DropdownMenu.tsx

```diff
@@ -40,11 +40,13 @@ export default class DropdownMenu extends React.Component<Props> {
     const pos = this.props.position
     const { innerHeight } = window
     const rect = menu.getBoundingClientRect()
+
     if (pos.bottom > innerHeight - rect.height) {
-      menu.style.bottom = "40px"
+      const bottom = rect.height + 40
+      menu.style.bottom = `${bottom}px`
       menu.style.top = "auto"
     } else {
-      menu.style.top = "40px"
+      menu.style.top = "-2px" // This is to hide the border of the buttonsContainer class
```

Comment:
Here i'm not sure other way to do it. The intention of set the top `-2px` is to move up the dropdown menu to fix this --> https://github.com/cultureamp/kaizen-design-system/pull/120#issuecomment-576544686

Reply:
I think that's how it was done before, so it should be fine. Thanks!